CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the AeroGear Android SDK
+
+The AeroGear Android SDK is part of the [AeroGear project](https://aerogear.org/), see the [Community Page](https://aerogear.org/community) for general guidelines for contributing to the project.
+
+This document details specifics for contributions to the Android SDK.
+
+## Issue tracker
+
+The tracking of issues for the AeroGear Android SDK is done in the [AeroGear Android Project](https://issues.jboss.org/projects/AGDROID/issues) in the [JBoss Developer JIRA](https://issues.jboss.org).
+
+See the [AeroGear JIRA Usage and Guidelines Guide](https://aerogear.org/docs/guides/JIRAUsage/) for information on how the issue tracker relates to contributions to this project.
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
```

Comment:
...you're contributing a new feature -> ...you're contributing _to_ a new feature ?

Reply:
I grabbed most of this text from the template we use for the feedhenry org, I think these are minor though: https://github.com/feedhenry/fh-metrics-client/blob/399b4655c3460132841481000f0241af60036249/CONTRIBUTING.md#asking-for-help

Reply:
Actually, both are correct so nevermind (https://english.stackexchange.com/a/177056).